packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/services/forms/FormWithTextfieldDescriptionProvider.java

```diff
@@ -116,6 +116,7 @@ private FormDescription createFormDescription() {
                 .valueExpression("aql:self.name")
                 .style(defaultTextStyle)
                 .conditionalStyles(isReadOnlyConditionalTextStyle)
+                .diagnosticsExpression("aql:if self.name->size() > 0 and self.name.at(1) = self.name.at(1).toUpper() then null else 'WARNING: name should start with upper case' endif")
```

Comment:
`.size()` not `->size()` otherwise it will create a collection with one value (`self.name` here)